.ci/jenkins/Jenkinsfile.e2e.cluster

```diff
@@ -2,9 +2,11 @@
 
 helper = null
 
-minikubeClusterPlatform = 'minikube'
+kindClusterPlatform = 'kind'
 openshiftClusterPlatform = 'openshift'
 
+kindLogsFolder = "/tmp/kind-logs"
```

Comment:
Seems unused, there are literal strings in relevant places.

Reply:
Should be used now, thanks